buildenv/jenkins/perfPipeline.groovy

```diff
@@ -38,49 +36,69 @@ params.each { param ->
     }
 }
 
-node ("ci.role.test&&hw.arch.x86&&sw.os.linux") {
+env.L2_Machine = "test-rhibmcloud-rhel9-x64-1" //grab and execute python scripts here
+
+node (env.L2_Machine) {
```

Comment:
Not an issue that is introduced by this PR. I just realized that we use 2 machines for one perf run.
- one is env.L2_Machine for process logs using python
- one is the actual perf machine

During the perf run, the L2_Machine is in idle, but cannot be used by other jobs. L2_Machine is only used before and after the actual perf run. Num of L2_Machine can be large, and the actual perf run can be very long. This needs to be improved in the future.